### DISCUSSION TITLE
Welcome to scipiper Discussions!

### DISCUSSION THREAD

**Discussion by aappling-usgs** [DOI-USGS/scipiper#149] [General]
## 👋 Welcome!
Let's try using Discussions as a place to raise questions and discuss hard pipelining challenges. GitHub recommends Discussions in place of Issues for such interactions.

You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by jsadler2** [comment]
Sounds cool! ... but what makes this any different than an Issue?

**Reply by jsadler2** [reply]
Ah. I can make a thread. 😎

**Reply by jsadler2** [reply]
But is that all that's different?

**Reply by aappling-usgs** [reply]
Maybe just a conceptual distinction between discussions (where what action to take is unclear) and issues (where there's a defined action that someone needs to take)? I'm new to this, just thought it'd be nice to try it out and see if we find benefits.

**Reply by aappling-usgs** [reply]
Also, click on Discussions in the top navigation bar and it takes you to thread sorting interface that differs from Issues. Maybe recent/active discussion threads are better at bubbling to the top. There's also a feature where you can mark one comment as the answer, which is probably more useful for large-scale software support, but maybe still fun for us sometimes. Anyhow, let's just see!

**Reply by jsadler2** [reply]
Nice! Yeah I'm starting to see the possibilities now. I do like the distinction of 

Discussion:
* no course of action (maybe just reporting results for my repos) and
* not sure of course of action

Issue:
* defined action that is assigned

**Reply by jordansread** [reply]
Cool, I can be part of the thread too!
